### PR TITLE
Add links to stats

### PR DESCRIPTION
**Pull request by TheOtterlord**
Add links to stats at bottom of contributor page.

### CODE REVIEW

File: src/components/achievements/Stat.astro [hunk 1]
```diff
@@ -25,17 +26,22 @@ const repoCounts = Object.entries(stat).sort(([, a], [, b]) => b - a);
...
           <li class="grid grid-cols-2 items-center">
-            <span
-              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words"
+            <a
+              href={`https://github.com/withastro/${repo}/${
+								type === 'issues' ? 'issues' : 'pulls'
+							}?q=is:${type === 'issues' ? 'issue' : 'pr'}+${
+								type === 'reviews' ? 'reviewed-by' : 'author'
+							}:${username}`}
+              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words hover:text-white"
```
Comment (delucis) [inline src/components/achievements/Stat.astro]: The Astro Tailwind kit includes a `link` class that should work well here. Lets us keep the text white but includes standard hover styles.
```suggestion
              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words link"
```

File: src/components/achievements/Stat.astro [hunk 2]
```diff
@@ -25,17 +26,22 @@ const repoCounts = Object.entries(stat).sort(([, a], [, b]) => b - a);
...
+  <ul class="text-[13px] leading-tight">
     {
       repoCounts.map(([repo, repoCount]) => {
         const percentage = (repoCount / maxCount) * 100;
         return (
           <li class="grid grid-cols-2 items-center">
-            <span
-              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words"
+            <a
+              href={`https://github.com/withastro/${repo}/${
```
Comment (delucis) [inline src/components/achievements/Stat.astro]: What do you think of moving the logic figuring out the URL up to the frontmatter where it's likely to be a bit more readable? If the ternary is a bit much, it's also easier to break things up there if you need to.

**Review by delucis — COMMENTED** [review]
Thank you for the PR @TheOtterlord! Works great in my testing. Really appreciate the idea and getting it wired up!

Left a couple of small comments but overall seems perfect 🙌

File: src/components/achievements/Stat.astro (hunk 2 above)
Comment (theotterlord) [reply src/components/achievements/Stat.astro]: Moved the query to frontmatter in https://github.com/delucis/astro-badge/pull/27/commits/1bc3060d8b67bdeb602b4f080d8e46f4547b890e

**Comment by TheOtterlord** [comment]
Thanks! I moved the query logic for the url to the frontmatter, and added the class.

**Review by delucis — APPROVED** [review]
This is such a nice contribution. Already enjoying browsing through what people have done.

Made one last quick change because I realised we already had the username in `<Stat>` but I hadn’t typed the `contributor` object very accurately.

Thanks again for this @TheOtterlord!

**Pull request merged**